app/main.py

```diff
@@ -1,3 +1,52 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", float]) -> "Distance":
```

Comment:
Int also possible.
And also remove quotes around class name. You need `from __future__ import annotations ` for this
Fix all over the project